feature/document/src/main/java/com/mifos/feature/document/document_dialog/DocumentDialogScreen.kt

```diff
@@ -0,0 +1,382 @@
+                            .fillMaxWidth()
+                            .padding(start = 16.dp, end = 16.dp),
+                        trailingIcon = {
+                            if (descriptionError) {
+                                Icon(imageVector = Icons.Filled.Error, contentDescription = null)
+                            }
+                        },
+                        enabled = false,
+                        maxLines = 1,
+                        colors = OutlinedTextFieldDefaults.colors(
+                            focusedBorderColor = if (isSystemInDarkTheme()) BluePrimaryDark else BluePrimary,
+                        ),
+                        textStyle = LocalDensity.current.run {
+                            TextStyle(fontSize = 18.sp)
+                        },
+                        keyboardOptions = KeyboardOptions(imeAction = ImeAction.Next),
+                        isError = fileError,
+                        supportingText = {
+                            if (fileError) {
+                                Text(
+                                    modifier = Modifier.fillMaxWidth(),
+                                    text = stringResource(id = R.string.feature_document_message_file_required),
+                                    color = MaterialTheme.colorScheme.error
+                                )
+                            }
+                        }
+                    )
+
+                    Spacer(modifier = Modifier.height(20.dp))
+
+                    Box(modifier = Modifier.padding(20.dp, 0.dp, 20.dp, 0.dp)) {
+                        Button(
+                            onClick = {
+                                openFilePicker.invoke()
+                            },
+                            shape = RoundedCornerShape(50.dp),
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .height(50.dp), 
+                            colors = ButtonColors(
+                                containerColor = BluePrimary,
+                                contentColor = White,
+                                disabledContainerColor = BluePrimary,
+                                disabledContentColor = White
+                            )
+                        ) {
+                            Text(text = "BROWSE")
+                        }
+                    }
+
+                    Spacer(modifier = Modifier.height(20.dp))
+
+                    Box(modifier = Modifier.padding(20.dp, 0.dp, 20.dp, 0.dp)) {
+                        Button(
+                            onClick = {
+                                  if(validateInput())
+                                  {
+                                      uploadDocument.invoke(name.text, description.text, dialogTitle)
+                                  }
+                            },
+                            shape = RoundedCornerShape(50.dp),
```

Comment:
Please correct me if I am wrong. Isn't by default the shape is `RoundedCornerShape(height.dp)`?